tests/conftest.py

```diff
@@ -363,3 +367,15 @@ def artifact_data():
     datacatalog = DataCatalog()
     datacatalog.from_predefined_catalogs("artifact_data")
     return datacatalog
+
+
+@pytest.fixture()
+def mock_model(tmpdir):
```

Comment:
There are already some other mocks for a Model. Not sure if you want to put this in conftest, as it will be different per test case.

Reply:
I'm using this fixture in test_grid_component as well. Should I use I put this fixture in both test_mesh_component.py and test_grid_component.py?